forms-flow-web/src/components/Form/List.js

```diff
@@ -608,9 +608,9 @@ const List = React.memo((props) => {
             {/*<img src="/form.svg" width="30" height="30" alt="form" />*/}
             <div className="flex-item-left">
               <div style={{ display: "flex" }}>
-                <h3 className="task-head" style={{ marginTop: "3px" }}>
+                <span className="task-head" style={{ marginTop: "3px" }}>
                   <i className="fa fa-wpforms" aria-hidden="true" />
-                </h3>
+                </span>
```

Comment:
Please check the hover colour sometime the span hover will be black

Reply:
checked , no issues